examples/llama/chat.py

```diff
@@ -0,0 +1,35 @@
+"""
+Demonstrates how to use the ChatInterface widget to create a chatbot using
+Llama2.
+
+Pre-requisites:
```

Comment:
I can wrap a try/except, but I don't think there's a way to include a huge llama model in GitHub (and it's probably undesired)

Reply:
I'm more thinking how we can simplify, document and/ or automate the process.

Reply:
This might work!
```python
from langchain import PromptTemplate, LLMChain
from langchain.llms import CTransformers

config = {"max_new_tokens": 2000, "temperature": 0.5}
llm = CTransformers(
    model="TheBloke/Llama-2-7b-Chat-GGUF",
    model_file="llama-2-7b-chat.Q5_K_M.gguf",
    config=config,
)

template = """<s>[INST] Help rephrase this sentence :
{question} [/INST] </s>
"""

#### Prompt
question_p = """I want to add a file source penguins.csv."""
prompt = PromptTemplate(template=template, input_variables=["question"])
llm_chain = LLMChain(prompt=prompt, llm=llm)
response = llm_chain.run({"question": question_p})
response
```